core/src/main/resources/defaults/blueprints/cdp-data-science.bp

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "CDP 1.0 Data Science template with Spark, Impala and Hive",
+  "description": "CDP 1.0 Data Science template with Spark, Impala, Hue and Hive",
```

Comment:
In general, I am not sure we indeed need such a Data Science BP. The [DistroX MVP doc|https://docs.google.com/document/d/17ldoDaDF_lFsFBtQNH8YxYzmPig6pnwJuVVJKRSXXmc/edit#heading=h.fxq8ldxisnts] explicitly mentions 2 cluster shapes: Data Engineering (Spark, MR, Hive and a lot of others) and Data Mart (Hue and Impala only, plus dependencies).

@akanto @keyki Please advise if this Data Science thing is indeed essential. If not, I suggest renaming it to Data Mart and removing anything not required (e.g. Spark and Hive).

Reply:
Turns out that Hue requires Hive.

Reply:
Okay, then it should be left there. 😃

Reply:
I think it might be a good idea to follow that naming what is in the MVP doc.

Reply:
One additional thing: 

If you rename a blueprint, unfortunately, the old one still stays there and not removed from Cloudbreak.

So I think we have two options:
- delete it with an SQL (it might not work if we have canary release)
- make sure that if you remove something from defaults, then also a DEFAULT label is removed from that blueprint (might have some performance implication)

@adoroszlai do you have maybe some recommendation to fix it?